README.md

```diff
@@ -0,0 +1,38 @@
+Hello Microverse
+This is my first project with microverse where I created 3 files; index.html, style.css, test.md and the last one needs to be ignored. The project is:
+
+Built With
+HTML
+CSS
+
+Getting Started
+Delete the testing section if the currect project doesn't require testing.
+
+To get a local copy up and running follow these simple example steps.
+
+Prerequisites
+Setup
+Install
+Usage
+Run tests
+Deployment
+Authors
+👤 Author1
+
+GitHub: @AnelisaDee
+Twitter: @AnelisaDee
+LinkedIn: AnelisaDee
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Hat tip to anyone whose code was used
+Inspiration
+etc
+📝 License
+This project is MIT licensed.
+
+NOTE: we recommend using the MIT license - you can set it up quickly by using templates available on GitHub. You can also use any other license if you wish.
```

Comment:
- [ ] KIndly make sure to clone the Microverse [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) in your repo in order to make the repo looks more professional and try to customize it according to your project. 

   **_Make sure your README looks more professional by defining the headers of each section. Kindly follow the provided template to achieve it_**